environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/endpoint/EnvironmentEndpoint.java

```diff
@@ -65,7 +65,7 @@
     @ApiOperation(value = EnvironmentOpDescription.DELETE_BY_NAME, produces = MediaType.APPLICATION_JSON,
             notes = ENVIRONMENT_NOTES, nickname = "deleteEnvironmentV1ByName")
     SimpleEnvironmentResponse deleteByName(@PathParam("name") String environmentName,
-        @QueryParam("cascading") @DefaultValue("true") boolean cascading,
+        @QueryParam("cascading") @DefaultValue("false") boolean cascading,
```

Comment:
if it does not affect the qaas side then I am fine with it

Reply:
qaas side is ready for this change
https://jira.cloudera.com/browse/QAINFRA-11810